Automattic/newspack-newsletters
# Prevent draft ads insertion

### All Submissions:






### Changes proposed in this Pull Request:



For some reason the 'post_status' param in WP_Query sometimes seems to be disregarded.

### How to test the changes in this Pull Request:

1. On `master`, create two ads - one published, one draft
2. Send a test newsletter - observe both ads included
3. Switch to this branch, send a test newsletter - observe only the published ad included

### Other information:
